packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -80,11 +94,75 @@ class TextSpan {
 
   /// A gesture recognizer that will receive events that hit this text span.
```

Comment:
Yeah. I'm also guessing we can't remove this one since I'm sure people have used it by now. It's an attractive API that does something people want to do. That's why I figured I should document it rather than remove it.